src/Uno.Sdk/Sdk/Sdk.props.buildschema.json

```diff
@@ -0,0 +1,170 @@
+{
+	"$schema": "https://raw.githubusercontent.com/mhutch/MonoDevelop.MSBuildEditor/main/MonoDevelop.MSBuild/Schemas/buildschema.json",
+	"properties": {
+		"UsingUnoSdk": {
+			"type": "bool",
+			"description": "Provides a flag to indicate that the Uno.Sdk is being used"
+		},
+		"UnoSingleProject": {
+			"type": "bool",
+			"description": "Indicates whether the source project should use UnoSingleProject features including support for the Platform specific directories"
+		},
+		"UnoVersion": {
+			"type": "nuget-version",
+			"description": "The version of the Uno.Sdk and the version of the core Uno packages to use."
+		},
+		"IsUnoHead": {
+			"type": "bool",
+			"description": "Indicates whether the project is a head project."
+		},
+		"AssetsFolder": {
+			"type": "folder-with-slash",
+			"description": "The folder containing the assets for this project."
+		},
+		"PlatformsProjectFolder": {
+			"type": "folder-with-slash",
+			"description": "The folder containing the platform-specific files for this project."
+		},
+		"AndroidProjectFolder": {
+			"type": "folder-with-slash",
+			"description": "The folder containing the Android-specific files for this project."
+		},
+		"iOSProjectFolder": {
+			"type": "folder-with-slash",
+			"description": "The folder containing the iOS-specific files for this project."
+		},
+		"MacCatalystProjectFolder": {
+			"type": "folder-with-slash",
+			"description": "The folder containing the Mac Catalyst-specific files for this project."
+		},
+		"MacOSProjectFolder": {
+			"type": "folder-with-slash",
+			"description": "The folder containing the macOS-specific files for this project."
+		},
+		"DesktopProjectFolder": {
+			"type": "folder-with-slash",
+			"description": "The folder containing the Skia Desktop-specific files for this project."
+		},
+		"WasmProjectFolder": {
+			"type": "folder-with-slash",
+			"description": "The folder containing the WebAssembly-specific files for this project."
+		},
+		"WindowsProjectFolder": {
+			"type": "folder-with-slash",
+			"description": "The folder containing the Windows-specific files for this project."
+		},
+		"DefaultLanguage": {
+			"defaultValue": "en",
+			"description": "The default language for the application.",
+			"type": "string"
+		}
+	},
+	"metadata": [
+		{
+			"$appliesTo": [
+				"UnoIcon",
+				"UnoImage",
+				"UnoSplashScreen"
+			],
+			"BaseSize": {
+				"type": "string",
+				"description": "Used to specify the size that will be used as a basement for the resize operations. e.g.: BaseSize=\"48,48\""
+			},
+			"Resize": {
+				"type": "bool",
+				"description": "Boolean value to say if the asset should Resized or not. By default just vectors asset are resized by default"
```

Comment:
```suggestion
				"description": "Boolean value to say if the asset should be Resized or not. By default, just vectors assets are resized by default"
```

Reply:
should we update the docs? Because this text is straight from the Resizetizer docs